media-proxy/src/libfabric_cq.c

```diff
@@ -119,23 +117,18 @@ static int rdma_poll_fd(int fd, int timeout)
     fds.fd = fd;
     fds.events = POLLIN;
     ret = poll(&fds, 1, timeout);
-    if (ret == -1) {
+    if (ret == !ret) {
```

Comment:
Always false, right?

Reply:
good catch, It suppose to be `if (!ret)`. Fixed.

Reply:
after further looking into it I was wrong. I removed my changes, the if case was ok before.